lib/plugin.js

```diff
@@ -30,43 +30,17 @@ export function addVisualRegressionTrackerPlugin(on, config) {
       }
       return null;
     },
-    ["VRT_TRACK_IMAGE_MULTIPART"]: async (props) => {
-      const data = multipartDtoToFormData({
-        ...props,
-        buildId: vrt.buildId,
-        projectId: vrt.projectId,
-        branchName: vrt.config.branchName,
-      });
-
-      const result = await vrt["submitTestRunMultipart"](data);
-
-      if (!props.keepScreenshot) {
-        unlinkSync(props.imagePath);
-      }
-      return result;
-    },
-    ["VRT_TRACK_BUFFER_MULTIPART"]: async (props) => {
-      const data = bufferDtoToFormData({
-        ...props,
-        buildId: vrt.buildId,
-        projectId: vrt.projectId,
-        branchName: vrt.config.branchName,
-      });
-
-      const result = await vrt["submitTestRunMultipart"](data);
-      return result;
-    },
-    ["VRT_TRACK_IMAGE_BASE64"]: async (props) => {
-      const result = await vrt["submitTestRunBase64"](props);
-      return result;
-    },
-    ["VRT_PROCESS_ERROR_RESULT"]: async (testRunResult) => {
+    ["VRT_TRACK"]: async (props) => {
       try {
-        vrt["processTestRun"](testRunResult);
+        return await vrt.track(props, props?.retryLimit);
       } catch (err) {
         return err.message ?? err;
       }
-      return null;
-    },
+      finally{
```

Comment:
check this part

Reply:
checked, screenshots are stored if we pass keepScreenshot: true and removed otherwise.